spec/unit_tests/moab/storage_repository_spec.rb

```diff
@@ -76,6 +76,61 @@
     expect(storage_repo.object_size('jq937jp0017')).to be_between(345_000, 346_000)
   end
 
+  context '#verify_no_nested_moabs' do
+    let(:path) { 'spec/fixtures/bad_root01/bad_moab_storage_trunk/xx/000/xx/0000/xx000xx0000' }
+    let(:verification_array) { storage_repo.verify_no_nested_moabs(path) }
+
+    it 'returns correct data structure' do
+      expect(storage_repo.verify_no_nested_moabs(path)).to be_kind_of Array
```

Comment:
maybe show that an item in the array is the expected structure, also?  That all array elements are the expected structure (allowed keys, etc.)